### ISSUE TITLE
Uncaught ReferenceError during processing SetDisplayLayout(displayLayout = DEFAULT) for navigation app

### ISSUE THREAD

**Issue by GetmanetsIrina** [smartdevicelink/sdl_hmi#608]
#### Description:
HMI Uncaught ReferenceError during processing SetDisplayLayout(displayLayout = DEFAULT) for navigation app

#### Precondition:

1. SDL and HMI are started
2. Navigation mobile app is registered and activated

#### Steps to reproduce:
1. App requests SetDisplayLayout(displayLayout = DEFAULT)

#### Exected result:
HMI responds with
{"success":true,"resultCode":"SUCCESS"}
HMI applies appropriate template

#### Actual result:
```
SDL -> HMI [10:10:5:566]: {"id":58,"jsonrpc":"2.0","method":"UI.SetDisplayLayout","params":{"appID":1516996278,"displayLayout":"DEFAULT"}}
UIRPC.js:160 FFW.UI.onRPCRequest
UIRPC.js:506 FFW.UI.SetDisplayLayoutResponse
UIRPC.js:520 Uncaught ReferenceError: NAV_FULLSCREEN_MAP is not defined
 at d.onRPCRequest (UIRPC.js:520)
 at d [as onRPCRequest] (ember-1.0.pre.min.js:14)
 at d.onWSMessage (RPCClient.js:190)
 at WebSocket.socket.onmessage (RPCClient.js:81)
```
#### Environment:
* Browser and Version: Chromium (Version 92.0.4515.159)
* SDL HMI Version: release/5.6.0 (ceb8b3c)
* Testing Against: SDL: release/8.0.0 (5da5770), SyncProxyTester(Version 20210729-Android (1082))

**Comment by GetmanetsIrina** [comment]
Please notice Luxoft team is going to prepare a fix.

**Comment by jacobkeeler** [comment]
Fixed via #615
